py_analyserver/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM socrata/py_nltk
+
+RUN pip install Flask>=0.10.1 \
```

Comment:
What version of pip are we running? This could become relevant if we depend on features that get deprecated.